Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -719,11 +730,15 @@ UnitValueMap: !dictionary
     minuten: 60
     minuut: 60
     min: 60
+    min.: 60
     secondes: 1
     seconden: 1
     seconde: 1
     secs: 1
     sec: 1
+    kwartier: 3600
```

Comment:
This seems incorrect. Especially as there are not the amounts of seconds in a half/quarter hour. Is this really necessary?

Reply:
it is necessary because 'kwartier' contains both the fraction 'quarter' and the unit 'hour', so its value in UnitValueMap is required, for example, by ParseNumberSpaceUnit in BaseDurationParser

Reply:
This seems strange, as kwartier would be a quarter of this value. Perhaps the best option would be for the code that consults the table to be aware of this?
I'm fine with keeping it for now, but please take a look to see if we can be more consistent.